### PR TITLE
Simple plugin mechanism

### PR DESCRIPTION
**Pull request by gberaudo**
Allow compiling application code with the OL3-Cesium library by simply
putting plugin javascript code in the src/plugins directory.

User automatically gets error checking, styling and documentation.

This extension mechanism is intended for people not familiar with
Closure. More advanced use-cases should be handled with a custom build.

### CODE REVIEW

File: src/plugins/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Extending the OL3-Cesium library classes requires both the library code and
+the extension code to be compiled together by Closure.
```
Comment (elemoine) [inline src/plugins/README.md]: `by Closure` -> `with Closure Compiler`.

File: src/plugins/.gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```
Comment (elemoine) [inline src/plugins/.gitignore]: What about the `README.md` file that you added?

Comment (gberaudo) [inline src/plugins/.gitignore]: Fixed.

**Comment by gberaudo** [comment]
@elemoine, do you have other comments?

**Comment by gberaudo** [comment]
Rebased on master.

**Comment by gberaudo** [comment]
Anymore review?

**Comment by elemoine** [comment]
Looks good to me. Thanks.

**Comment by gberaudo** [comment]
Thank you for the review @elemoine.

**Pull request merged**